macros/schema_tests/equal_sum.sql

```diff
@@ -0,0 +1,34 @@
+{% macro test_equal_sum(model) %}
```

Comment:
I think you'll want to make the signature here:
```
{% macro test_equal_sum(model, column_name, compare_model, compare_column) %}
```

Then you can get rid of the `kwargs` lines below. We needed to do the `kwargs` trick to make schema tests compatible with the `v1` and `v2` schema.yml specs, but I don't think we want to support the v1 spec for net-new tests.